src/libstore/local-store.cc

```diff
@@ -59,6 +59,8 @@ struct LocalStore::State::Stmts {
     SQLiteStmt QueryAllRealisedOutputs;
     SQLiteStmt QueryPathFromHashPart;
     SQLiteStmt QueryValidPaths;
+    SQLiteStmt QueryRealisationRealisationReferences;
+    SQLiteStmt AddRealisationRealisationReference;
```

Comment:
It is indeed. That was because there used to be a `RealisationPathReference`, but it’s actually not needed. Fixed in dcabb461242e5c38e0a2780dc5869b91c13241d6